src/Misc/tools.ts

```diff
@@ -15,6 +15,7 @@ import { PromisePolyfill } from './promise';
 import { TimingTools } from './timingTools';
 import { InstantiationTools } from './instantiationTools';
 import { GUID } from './guid';
+import { IScreenshotSize } from './screenshotTools';

```

Comment:
The only problem is that we do not want to import the file automatically for size reason (treeshacking). Do you mind moving the interface in its own file?